.gitignore

```diff
@@ -6,6 +6,7 @@ results.xml
 driver/examples/*.json
 driver/examples/**/*.json
 driver/examples/comm
+20*-*-*-*-*-*.json
```

Comment:
Not for your PR, but we may want to reduce side-effect files of running the model or at least make the location these go to configurable.